### DISCUSSION TITLE
[Android] Resume/pause animation

### DISCUSSION THREAD

**Discussion by Ncit** [google/filament#4732] [Q&A] [answered]
Hello there, I did not find any clue as to how to pause/resume in built animation in glft file. Can someone help me?

**Comment by bejado** [comment, marked as answer]
Take a look at the [`Animator.applyAnimation`](https://github.com/google/filament/blob/main/libs/gltfio/include/gltfio/Animator.h#L48) API. It takes a time in seconds, it's up to the app to supply play/pause logic.
